install/migrations/update_10.0.x_to_10.1.0/notifications.php

```diff
@@ -288,3 +288,107 @@
     ]);
 }
 /** /Add new notification for AutoBump */
+
+/** Add new notification for New document */
+if (countElementsInTable('glpi_notifications', ['itemtype' => 'Ticket', 'event' => 'add_document']) === 0) {
+    $DB->insertOrDie('glpi_notificationtemplates', [
```

Comment:
Reading the migration, I wondered if it was really worth adding this default notification template/translation.
Indeed, people that already customize their templates will probably want to use their existing templates, or to create a new dedicated template based on an existing one.
In conclusion, I think we should only add the base notification information (itemtype/event + targets), but not associate any template to it.

@orthagh What do you think about it?

Note: It will probably mean that we will have to exclude `glpi_notification*` tables from the `\Glpi\Tests\Command\TestUpdatedDataCommand` command, as updated data will not anymore contains same number of notifications as fresh install data, but it does not seems a problem to me.

Reply:
On update process, there shouldn't be a single notification added. Nor template, nor base notification.
We shouldn't mess with existing instance notifications.